README.md

```diff
@@ -17,8 +17,27 @@ All other tutorials assume that you have already completed that tutorial and are
 ## Getting Started
 
 Step one is cloning this repo.
-If you're looking to quickly get started with a tutorial, we recommend using our [Docker setup](#docker).
-If you want to install things yourself, you can follow our [installation steps](#install) below instead.
+
+```bash
+git clone https://github.com/snorkel-team/snorkel-tutorials.git
+cd snorkel-tutorials
+```
+
+As with Snorkel, our tutorials require Python 3.6+.
+If you're looking to quickly get started with a tutorial, we recommend using
+our [Docker setup](#docker).
+If you want to install things yourself using `pip` or `conda`, you can follow
+our [installation steps](#install) below instead.
+
+*A quick note for Windows users*
+
+If you're using Windows, we highly recommend using the [Docker setup](#docker)
+or the [Linux subsystem](https://docs.microsoft.com/en-us/windows/wsl/faq).
```

Comment:
I think A. tried the Linux subsystem, and it didn't work so hot—but certainly better than whatever other defaults

Reply:
had to do with particularities of conda install...can't defend against everything (but Docker can)